packages/veritone-react-common/src/components/SourceManagement/NullState/styles.scss

```diff
@@ -0,0 +1,40 @@
+.nullStateView {
```

Comment:
The majority of these styles seem unnecessary.

I've listed the ones below that are actually doing something, the rest you can get rid of.

```
padding-top: 40vh;
display: flex;
flex-direction: column;
align-items: center;
```

Reply:
fixed